src/pages/TrueSight/components/FilterBar/index.tsx

```diff
@@ -2,42 +2,72 @@ import React, { useState } from 'react'
 import { t, Trans } from '@lingui/macro'
 import { Flex } from 'rebass'
 import { useMedia } from 'react-use'
+import styled from 'styled-components'
 
 import {
   TextTooltip,
   TrueSightFilterBarLayout,
   TrueSightFilterBarLayoutMobile,
   TrueSightFilterBarSection,
+  ButtonText,
+  UnSubscribeButton,
+  StyledSpinner,
 } from 'pages/TrueSight/styled'
-import { TrueSightFilter, TrueSightSortSettings, TrueSightTabs, TrueSightTimeframe } from 'pages/TrueSight/index'
+import { TrueSightFilter, TrueSightTabs, TrueSightTimeframe } from 'pages/TrueSight/index'
 import TimeframePicker from 'pages/TrueSight/components/FilterBar/TimeframePicker'
 import TrueSightToggle from 'pages/TrueSight/components/FilterBar/TrueSightToggle'
 import useParsedQueryString from 'hooks/useParsedQueryString'
-import TrueSightSearchBox from 'pages/TrueSight/components/FilterBar/TrueSightSearchBox'
+import TrueSightSearchBox, { SelectedOption } from 'pages/TrueSight/components/FilterBar/TrueSightSearchBox'
 import NetworkSelect from 'pages/TrueSight/components/FilterBar/NetworkSelect'
 import useGetTokensForSearchBox from 'pages/TrueSight/hooks/useGetTokensForSearchBox'
 import useDebounce from 'hooks/useDebounce'
 import useGetTagsForSearchBox from 'pages/TrueSight/hooks/useGetTagsForSearchBox'
-import useTheme from 'hooks/useTheme'
 import { MouseoverTooltip } from 'components/Tooltip'
-import { ButtonEmpty } from 'components/Button'
-import { BarChart } from 'react-feather'
-import { useTrendingSoonSortingModalToggle } from 'state/application/hooks'
-import ModalSorting from 'pages/TrueSight/components/FilterBar/ModalSorting'
+import { Container as SearchContainer, Input as SearchInput } from 'components/Search'
+import NotificationIcon from 'components/Icons/NotificationIcon'
+import useTheme from 'hooks/useTheme'
+import { useTrueSightUnsubscribeModalToggle } from 'state/application/hooks'
+import useNotification from 'pages/TrueSight/hooks/useNotification'
+import UnsubscribeModal from 'pages/TrueSight/components/UnsubscribeModal'
+
+const TrueSightSearchBoxOnMobile = styled(TrueSightSearchBox)`
```

Comment:
To extend `TrueSightSearchBox`, there must be className props inside TrueSightSearchBox